src/style.css

```diff
@@ -1 +1,73 @@
-/* add styles here */
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  font-style: normal;
+  font-weight: normal;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-style: bold;
+  font-weight: bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.small-search-bar {
+  margin-top: 20px;
```

Comment:
The same comment about margins applies here. It's better to be consistent and only add top or bottom margins, but not both.